0-ad-history-mod-package/simulation/templates/units/han/hero_wei_qing.xml

```diff
@@ -15,6 +15,12 @@
     <Civ>han</Civ>
     <SpecificName>Wèi Qīng</SpecificName>
     <Icon>units/han/hero_wei_qing.png</Icon>
+    <History>
+        Wei Qin's early childhood was unfortunately not a happy one. He was born into a poor slave family and heavily mistreated and abused by his father. So much so, in fact, that at the age of only around ten, he couldn't stand it anymore and left. Wei Qing eventually found his biological mother at the palace of Princess Pingyang and started working there. And this time luck was on his side: when Emperor Wu himself once visited the palace, he was impressed by Wei Qing's half-sister, Wei Zifu, and decided to take her to his imperial palace as a concubine. Wei Qing, as her brother, was taken there as well and got employed as a palace guard. However, for this lucky twist of fate, he almost paid the full price: when Wei Zifu got pregnant with the emperor's first son, the empress got jealous. And for being her brother, Wei Qing was kidnapped and almost beaten to death if it weren't for his friends to come just in time for rescue. It wasn't all bad, though. As punishment for this crime, Emperor Wu promoted Wei Qing and Wei Zifu.
```

Comment:
Biological mother? Who was Wei Qin with beforehand? Adopted slave parents?

Reply:
An "illegitimate" child, born from a  different woman than his father's wife. This is also one of the reasons why he was treated so badly.

Reply:
OK, I can suggest a change that incorporates a bit more of that information.